### ISSUE TITLE
Pytorch 0.4.0 documentation typo for batchnorm's momentum parameter

### ISSUE THREAD

**Issue by bado-lee** [pytorch/pytorch#5625]
Hi, I found a typo in the pytorch 0.4.0's Batchnorm documentation.
http://pytorch.org/docs/master/nn.html?highlight=batchnorm2d#torch.nn.BatchNorm2d

Different from 0.3.x below equation explanation is added

![image](https://user-images.githubusercontent.com/26222919/37128888-88de89d4-22bf-11e8-9153-1fbec0f6be0c.png)

but, below term should be 

![image](https://user-images.githubusercontent.com/26222919/37128902-abdb3950-22bf-11e8-8f14-d82db4b3b0be.png)

corrected as

![image](https://user-images.githubusercontent.com/26222919/37129017-3974a756-22c0-11e8-837b-64b6e1a541ac.png)

Since it is previous step's statistics term.
So, "new" should be removed or be replaced as "old".

This typo is in every box within BatchNorm1d, 2d etc...

**Comment by soumith** [comment]
closed via https://github.com/pytorch/pytorch/pull/5450/
